code/modules/clothing/under/jobs/Plasmaman/civilian_service.dm

```diff
@@ -0,0 +1,63 @@
+//obj/item/clothing/under/plasmaman/civilian
+//	name = "civilian plasma envirosuit"
+//	desc = "A special containment suit that allows plasma-based lifeforms to exist safely in an oxygenated environment, and automatically extinguishes them in a crisis. Despite being airtight, it's not spaceworthy. It is painted grey."
+//	icon_state = "plasmaman_civilian"
+//	item_state = "plasmaman_civilian"
+//	item_color = "plasmaman_civilian"
```

Comment:
ded code?

Reply:
oh yeah that was when i was testing civ suits, im removing this